api/test/robot_tests/plugin_manager/plugin_manager.robot

```diff
@@ -109,7 +109,7 @@ POST /plugins-manager/{id}/execute already started
 GET /plugin-manager/{id} with invalid id
     ${response}=    GET    ${API_URL}/plugin-manager/invalid    expected_status=${STATUS_NOT_FOUND}
     Should Be Equal As Strings    ${response.json()['code']}    Plugin-0001
-    Should Be Equal As Strings    ${response.json()['message']}    get plugin error: no plugin matching id invalid
+    Should Be Equal As Strings    ${response.json()['message']}    get plugin error: no plugin matching correlationID invalid
```

Comment:
Are we sure this message is correct?
`no plugin matching correlationID invalid` 
TBH I haven't got the point! 🤔

Reply:
It is correct: https://github.com/massalabs/thyra/blob/main/pkg/plugin/manager.go#L114

Reply:
`no plugin matching correlationID` it's clear, our is not!

no plugin matching correlationID **invalid** For me it seems weird the invalid in the end of the sentence.

Reply:
The ID we are trying to get is literally `invalid`, that's why it is displayed like this. 
If the ID was "a8e3a9f2be02", "get plugin error: no plugin matching correlationID a8e3a9f2be02" would be returned.

Reply:
Got it! It's an interpolation on the **correlationID**!

Thank you, Thomas!